data/fr/maps/level_0/dialog_1.txt

```diff
@@ -0,0 +1,6 @@
+Jist
+
+Salut, aventuriers ! La situation est critique. Des brigands ont dérobé des ossements d'ogres enfouis dans la crypte.
+Depuis, le village est constamment attaqué par les morts-vivants…
+Heureusement, j'ai réussi à récupérer les ossements.
```

Comment:
One line is missing:
`You must take them back to their place in the crypt. The necropolis is far to the west of here.`

I would translate it this way:
`Vous devez les ramener à leur emplacement d'origine au sein de la crypte. La nécropole est loin à l'ouest d'ici.`